src/plugins/kv-analytics-plugin.js

```diff
@@ -10,28 +10,38 @@ export default Vue => {
 			snowplowLoaded = inBrowser && typeof window.snowplow === 'function';
 		},
 		pageview: (to, from) => {
-			const toUrl = window.location.origin + to.path;
-			const asyncFromUrl = window.location.origin + from.path;
+			if (!inBrowser) return false;
+
+			let toUrl = window.location.href;
+			let fromUrl = document.referrer;
```

Comment:
Exit if not in the browser... or set default to + from url properties for use on server rendered pageview events.